extensions/brew/src/components/actionPanels.tsx

```diff
@@ -57,8 +57,12 @@ export function CaskActionPanel(props: {
           <Action.ShowInFinder path={brewInstallPath(cask)} />
         </ActionPanel.Section>
         <ActionPanel.Section>
-          <Action.OpenInBrowser url={cask.homepage} />
-          <Action.CopyToClipboard title="Copy URL" content={cask.homepage} />
+          <Action.OpenInBrowser url={`https://formulae.brew.sh/cask/${cask.token}`} title="Open Cask" />
+          <Action.CopyToClipboard title="Copy Cask URL" content={`https://formulae.brew.sh/cask/${cask.token}`} />
+        </ActionPanel.Section>
+        <ActionPanel.Section>
+          <Action.OpenInBrowser url={cask.homepage} title="Open Homepage" />
+          <Action.CopyToClipboard title="Copy Homepage URL" content={cask.homepage} />
```

Comment:
A small suggestion: Would you mind moving those `title` fields to the front of the `url` field? This could make all props' orders look consistent.

And ditto for the rest of the code.